spec/unit/lib/mail_adapters/mailgun_spec.rb

```diff
@@ -26,67 +26,6 @@
     stub_const('Howitzer::Email::SUBJECT', message_subject)
   end
 
-  describe '.find' do
```

Comment:
we need to rewrite these tests. Instead of testing "find" method, we need to test "find_by_recipient"